packages/editor-ui/src/store.ts

```diff
@@ -774,16 +779,32 @@ export const store = new Vuex.Store({
 		allNodeTypes: (state): INodeTypeDescription[] => {
 			return state.nodeTypes;
 		},
-		nodeType: (state) => (nodeType: string): INodeTypeDescription | null => {
+		allNodeTypesDefaultVersion: (state): INodeTypeDescription[] => {
+			return state.nodeTypesDefaultVersion;
+		},
+		nodeType: (state, getters) => (nodeType: string, typeVersion?: number): INodeTypeDescription | null => {
+			if(typeVersion === undefined) {
+				typeVersion = getters.nodeTypeDefaultVersion(nodeType) || 1;
+			}
 			const foundType = state.nodeTypes.find(typeData => {
-				return typeData.name === nodeType;
+				return typeData.name === nodeType && typeData.version === typeVersion;
 			});
 
 			if (foundType === undefined) {
 				return null;
 			}
 			return foundType;
 		},
+		nodeTypeDefaultVersion: (state) => (nodeType: string): number | null => {
+			const foundType = state.nodeTypesDefaultVersion.find(typeData => {
+				return typeData.name === nodeType;
+			});
+
+			if (foundType === undefined) {
+				return null;
+			}
+			return foundType.version;
```

Comment:
nitpick - tertiary condition is clearer here 
```suggestion
			return foundType? foundType.version : null;
```